tools/trap/HowToUseTrap.md

```diff
@@ -3,11 +3,15 @@ Here we explain how to enable build support for DUMP upload,
 and how to actually upload and parse it in the event of a crash during runtime.
 
 ## Contents
+> [Prerequisites](#prerequisites)  
 > [Build Steps](#how-to-enable-memory-dumps)  
 > [Parsing Steps](#how-to-parse-RAMDUMP)  
 > [Upload Steps](#how-to-upload-RAMDUMP-or-UserfsDUMP)  
 > [Porting Guide](#how-to-port-memory-dump-functionality)  
 
+## Prerequisites
+Install Python 3.0 or above.
```

Comment:
User must have Python 3.0 or above installed?

Reply:
Python 2 is deprecated. I think all user should have python 3

Reply:
>  junmin-kim 3 minutes ago
Python 3.6 reached end-of-life.
And in Ubuntu 20.04 LTS, Python 3.8 is included.
How about mandate Python 3.8 or above in prerequisites?

Could you check this?

Reply:
Umm, once we verify F strings with Python 3.7 , I will update it to 3.7